### ISSUE TITLE
Profiles Editor: main configuration layout is hard to parse

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#218117]
Testing #217139

It feels like all the right ingredients are here, but it's hard to understand exactly what's happening in this layout.
- "Copy from" is somewhat vague since all the user has seen at this point is New Profile -> From Template. This might be where a more typical form layout with a description (things settings UI) could help here
- The table itself is also missing some context/information. Without a label/desc, I don't know if it's required of me as a user to interact with it
- First column is missing a column header.
- The Copy (Python) formatting feels a bit strange. "Copy from Python" might feel more natural

![CleanShot 2024-06-25 at 09 54 43@2x](https://github.com/microsoft/vscode/assets/25163139/24266643-9b9a-4486-ae21-49b6be081269)

**Comment by sandy081** [comment]
It seems this issue https://github.com/microsoft/vscode/issues/218162 has a sketch of improvements. Let's continue the discussion there.
